docs/faang_experiment_metadata.md

```diff
@@ -40,6 +40,7 @@ These following elements must always be present in any experiment metadata
     * RNA-seq of non coding RNA
     * transcription profiling by high throughput sequencing
     * WGS
+    * CAGE-Seq
```

Comment:
For consistency please use lower case s for 'seq' in all places

Reply:
done in f6a8d89